Dockerfile

```diff
@@ -3,8 +3,24 @@ ARG FROM_TAG
 FROM debian:$FROM_TAG
 
 RUN \
-    sed -e 's/^Components: main$/& contrib/g' -i /etc/apt/sources.list.d/debian.sources && \
     apt-get update && \
-    apt-get install --yes --no-install-recommends zfsutils-linux && \
+    apt-get install build-essential autoconf automake libtool gawk alien fakeroot dkms libblkid-dev uuid-dev libudev-dev libssl-dev zlib1g-dev libaio-dev libattr1-dev libelf-dev python3 python3-dev python3-setuptools python3-cffi libffi-dev python3-packaging debhelper-compat dh-python po-debconf python3-all-dev python3-sphinx libpam0g-dev libcurl4-openssl-dev linux-headers-generic wget dh-sequence-dkms --yes && \
+    mkdir -p /opt/zfs && \
+    cd /opt/zfs && \
+    wget https://github.com/openzfs/zfs/archive/refs/tags/zfs-2.2.3.tar.gz && \
+    tar -zxf zfs-2.2.3.tar.gz && \
+    cd zfs-zfs-2.2.3 && \
+    wget https://patch-diff.githubusercontent.com/raw/openzfs/zfs/pull/15784.diff && \
```

Comment:
The patch is required for `arm64` architecture, but it does not harm for `amd64`